### PR TITLE
Fix #460 Doc update for proxy support

### LINKED ISSUE #460: Doc update for proxy support
Recently we did some work from adb-utils (https://github.com/projectatomic/adb-utils/pull/150) side to support proxy but we also need to update our documents to make sure user understand how to use this feature.

### PR DESCRIPTION
**Pull request by praveenkumar**
This patch will provide a basic documentation for proxy support.

### CODE REVIEW

File: docs/using.rst [hunk 1]
```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
 .. _Docker Tooling: http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php
       **Note:** Testing has been done with Eclipse 4.5.0.
 
+Using the ADB behind proxy
+==========================
+
+Today, Most of organizations have proxy enabled for internal network and good news is ADB can be setup behind proxy. All you need to do is un-comment proxy variable in the Vagrantfile and assign required value.
+
```
Comment (Preeticp) [inline docs/using.rst:188]: Can we make a few small changes?
Today, most of the organizations enable proxy servers for internal networks, the good news is, ADB can be set up behind this proxy. All you need to do is, un-comment the proxy variable in the Vagrantfile and assign the required value.

File: docs/using.rst [hunk 2]
```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
...
+**Note:** Only HTTP and HTTPS proxy currently supported.
+
+Variable available for proxy setup::
+
```
Comment (Preeticp) [inline docs/using.rst:192]: Minor changes:
Note: Currently, only the HTTP and HTTPS proxy are supported.

The variable available for proxy setup is as shown below::

File: docs/using.rst [hunk 3]
```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
 .. _Docker Tooling: http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php
       **Note:** Testing has been done with Eclipse 4.5.0.
 
+Using the ADB behind proxy
```
Comment (LalatenduMohanty) [inline docs/using.rst]: We should mention http proxy in place of proxy.

File: docs/using.rst [hunk 4]
```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
...
+==========================
+
+Today, Most of organizations have proxy enabled for internal network and good news is ADB can be setup behind proxy. All you need to do is un-comment proxy variable in the Vagrantfile and assign required value.
```
Comment (LalatenduMohanty) [inline docs/using.rst]: s/ Today, Most of organizations have proxy enabled for internal network and good news is ADB can be setup behind proxy/ADB can be setup behind networks where proxy setup is required to access internet/

Comment (praveenkumar) [inline docs/using.rst]: I liked what @Preeticp suggested for this :)

File: docs/using.rst (hunk 3 above)
Comment (praveenkumar) [inline docs/using.rst]: done

File: docs/using.rst [hunk 5]
```diff
@@ -181,6 +181,19 @@ To use ADB with Host-Based tools
 .. _Docker Tooling: http://www.eclipse.org/community/eclipse_newsletter/2015/june/article3.php
       **Note:** Testing has been done with Eclipse 4.5.0.
 
+Using the ADB behind http proxy
+===============================
+
+Today, most of the organizations enable proxy servers for internal networks, the good news is, ADB can be set up behind this proxy. All you need to do is, un-comment the proxy variable in the Vagrantfile and assign the required value. 
+
```
Comment (LalatenduMohanty) [inline docs/using.rst:188]: "Today, most of the organizations enable proxy servers for internal networks, the good news is," -> I think we should avoid the term "Today" as it kind of refers to a certain time line. But what we are trying to communicate is a generic thing and the rest of the sentence is not necessary as this seems like extra information wrt to how many organizations are using proxy. Also the witting style should be nearly in sync with other documentation above and below it.

**Comment by LalatenduMohanty** [comment]
LGTM . But we should merge this after #482 as https://github.com/projectatomic/adb-atomic-developer-bundle/pull/482/ completes this PR.

**Comment by praveenkumar** [comment]
#482 already merged so merging this.

**Pull request merged**